pkgs/development/python-modules/mariadb/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, libmysqlclient, lib }:
+buildPythonPackage rec {
+  pname = "mariadb";
+  version = "1.0.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-S3sbMHowo8sSJy2bHcoNE7Zr09QVtKzrCkB/CYPzQaM=";
+    extension = "zip";
+  };
+
+  nativeBuildInputs = [
```

Comment:
```suggestion
  buildInputs = [
```

Reply:
Command `mariadb_config` is required in `setuptoolsBuildPhase`,
If I changed it to `buildInputs`, it cannot build...